lib/restaurant.rb

```diff
@@ -0,0 +1,37 @@
+require './lib/menu'
```

Comment:
Really nice structuring, with Restaurant class bringing together Order and Menu classes.